### ISSUE TITLE
[Question] Soft positive label or label smoothing

### ISSUE THREAD

**Issue by xuelicheng1992** [MIC-DKFZ/nnDetection#210]
### :question: Question

Hello,
I am considering experimenting with the use of soft labels or label smoothing. Because the possibility of some labels in my training data not being strictly correct。
Please tell me which part of the code I need to make modifications.

**Comment by mibaumgartner** [comment]
Soft labels are not supported in nnDetection and would require significant changes in the planning and evaluations stage. It might be possible to adapt the IO (dataloader) classs to manually load softlabels besides the normal labels for training purposes. There only the ptmodule would need some updates.

Using label smoothing should be straight forward since only a modification in the loss is necessary.

Note however that wrong labels have more repercussions than one might think, e.g. focal loss and hard negative mining will apply high weights to false negative predictions. Ensuring the highest possible label quality is essential to guarantee stable training and achieving the best possible results. 

Best,
Michael

**Comment by xuelicheng1992** [comment]
Thank you for the reply!
